packages/inter-protocol/src/vpool-xyk-amm/addPool.js

```diff
@@ -90,51 +93,31 @@ export const makeAddIssuer = (
 };
 
 /**
- * @param {ZCF<import('./multipoolMarketMaker.js').AMMTerms>} zcf
- * @param {(brand: Brand, pool: PoolFacets) => void} initPool add new pool to store
- * @param {Brand} centralBrand
- * @param {ERef<Timer>} timer
- * @param {IssuerKit} quoteIssuerKit
- * @param {import('./multipoolMarketMaker.js').AMMParamGetters} params retrieve governed params
- * @param {ZCFSeat} protocolSeat seat that holds collected fees
- * @param {WeakStore<Brand,ZCFMint>} brandToLiquidityMint
+ * @param {Baggage} baggage
+ * @param {import('./multipoolMarketMaker.js').AmmState} ammState
  * @param {(secondaryBrand: Brand, reserveLiquidityTokenSeat: ZCFSeat, liquidityKeyword: Keyword) => Promise<void>} onOfferHandled
  * @param {ERef<StorageNode>} storageNode
  * @param {ERef<Marshaller>} marshaller
  */
 export const makeAddPoolInvitation = (
-  zcf,
-  initPool,
-  centralBrand,
-  timer,
-  quoteIssuerKit,
-  params,
-  protocolSeat,
-  brandToLiquidityMint,
+  baggage,
+  ammState,
   onOfferHandled,
   storageNode,
   marshaller,
 ) => {
-  const makePool = definePoolKind(
-    zcf,
-    centralBrand,
-    timer,
-    quoteIssuerKit,
-    params,
-    protocolSeat,
-    storageNode,
-    marshaller,
-  );
+  const zcf = ammState.zcf;
```

Comment:
nit,
```suggestion
  const { zcf } = ammState;
```

Reply:
done. I noticed that I was doing that as I was refactoring, and mislaid the mental note to repair it.